src/wallet.ts

```diff
@@ -276,7 +277,7 @@ WalletMiddlewareOptions): JsonRpcMiddleware<any, Block> {
     const params = req.params as [string, string];
 
     const address = await validateAndNormalizeKeyholder(params[0], req);
-    const message = params[1];
+    const message = normalizeTypedMessage(params[1]);
```

Comment:
What is the material effect of this change for this middleware? I know that we have tests for this function in particular but should we add a test for this in `wallet.test.ts` to ensure that the behavior we want is being applied?

Reply:
Yeah, a simple test is fine, no need to test edge cases. Thanks :)